pom.xml

```diff
@@ -53,7 +53,8 @@
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.28</dep.slice.version>
         <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
-        <dep.tempto.version>1.23</dep.tempto.version>
+        <dep.tempto.version>1.24-SNAPSHOT</dep.tempto.version>
```

Comment:
Yes, it requires changes since tempto uses some TestNG internals: https://github.com/prestodb/tempto/commit/c7703669654203997a1776105665673890b0a22c